docs/docsite/rst/conf.py

```diff
@@ -32,7 +32,21 @@
 # the repository version needs to be the one that is loaded:
 sys.path.insert(0, os.path.abspath(os.path.join('..', '..', '..', 'lib')))
 
-VERSION = '2.15_ja'
+KNOWN_TAGS = {'all', 'ansible', 'core', 'core_lang', '2.10'}
+
+applied_tags_count = sum(
+    int(tags.has(known_tag_name)) for known_tag_name in KNOWN_TAGS
+)
+assert applied_tags_count == 1, (
+    'Exactly one of the following tags expected: {", ".join(tags)}'.
+    format(tags=KNOWN_TAGS)
+)
+
+VERSION = (
+    'devel' if tags.has('core_lang') or tags.has('core') or tags.has('ansible') or tags.has('all') else
```

Comment:
@webknjaz Hey, just looking at this backport for 2.15 and I believe we'll need to make a change here based on these two lines:

https://github.com/ansible/ansible-documentation/blob/235d12223535115ba732ff5f847e6999bf2a941e/docs/docsite/sphinx_conf/ansible_conf.py#L35 and https://github.com/ansible/ansible-documentation/blob/235d12223535115ba732ff5f847e6999bf2a941e/docs/docsite/sphinx_conf/core_conf.py#L35

```
'2.15' if tags.has('core_lang') or tags.has('core') else
'8' if tags.has('ansible') or tags.has('all') else
'2.10' if tags.has('2.10')
```

Does that look correct to you? I'll put this up on a test build but would appreciate your input. Thanks.